modules/javafx.controls/src/main/java/javafx/scene/chart/XYChart.java

```diff
@@ -1212,8 +1261,14 @@ public StyleableProperty<Boolean> getStyleableProperty(XYChart<?,?> node) {
         private boolean setToRemove = false;
         /** The series this data belongs to */
         private Series<X,Y> series;
+        private ObjectProperty<Series<X, Y>> seriesProperty = new SimpleObjectProperty<>();
         void setSeries(Series<X,Y> series) {
             this.series = series;
+            this.seriesProperty.set(series);
+        }
+
+        Series<X, Y> getSeries() {
+            return seriesProperty.get();
```

Comment:
This method is unused, so should probably be removed.

Reply:
Done.